converters/fromZigbee.js

```diff
@@ -5165,40 +5165,54 @@ const converters = {
         cluster: 'msCO2',
         type: 'readResponse',
         convert: (model, msg, publish, options, meta) => {
-            return {
-                led_feedback: ['OFF', 'ON'][msg.data[0x0203.toString()]],
-                enable_abc: ['OFF', 'ON'][msg.data[0x0202.toString()]],
-                threshold1: msg.data[0x0204.toString()],
-                threshold2: msg.data[0x0205.toString()],
-            };
+            const result = {};
+            if (msg.data.hasOwnProperty(0x0203)) {
```

Comment:
Are you sure that you can pass the number here? Property names as strong as ng values